data/src/main/java/com/anytypeio/anytype/data/auth/repo/block/BlockDataRepository.kt

```diff
@@ -922,4 +922,23 @@ class BlockDataRepository(
             identity = identity
         )
     }
+
+    override suspend fun removeSpaceMembers(space: SpaceId, identities: List<Id>) {
```

Comment:
My suggestion in the data module is to use 'sealed class Command'  to pass parameters, in this case you need to change or add only in one place, instead of 6.